src/components/VideoPlayer/BaseVideoPlayer.js

```diff
@@ -134,6 +136,17 @@ function BaseVideoPlayer({
     const handleFullscreenUpdate = useCallback(
         (e) => {
             onFullscreenUpdate(e);
+
+            if (e.fullscreenUpdate === VideoFullscreenUpdate.PLAYER_DID_DISMISS) {
+                currentVideoPlayerRef.current.getStatusAsync().then((status) => {
+                    if (status.volume > 0 && !status.isMuted) {
+                        return;
+                    }
+
+                    updateVolume(0);
```

Comment:
@shahinyan11 If we set the updateVolume to status.volume, the volume value can be synced between fullscreen and mini player, right? like below.
```diff
+if(status.value === 0 || status.isMuted) {
+     updateVolume(0);
+     return;
+}
+     updateVolume(status.volume);
```

Reply:
What about `updateVolume(status.value === 0 || status.isMuted ? 0 : status.volume);` ?